app/src/lib/git/core.ts

```diff
@@ -323,6 +326,15 @@ function getDescriptionForError(error: DugiteError): string | null {
   }
 
   switch (error) {
+    case DugiteError.BadConfigValue:
+      const errorEntry = Object.entries(DugiteErrorRegexes).find(
+        ([_, v]) => v === DugiteError.BadConfigValue
+      )
+      const m = stderr.match(errorEntry![0])
+      const value = m![1]
+      const key = m![2]
+
+      return `Unsupported value '${value}' for git config key '${key}'`
```

Comment:
This is also kind of meh… I'm pondering whether it's cleaner to just expose from dugite a `parseBadConfigValueInfo` function and then hide both this logic and the need to expose `GitErrorRegexes`. Thoughts?

Reply:
Yeah. I like the idea. 

Other thoughts to this area: We could have also have some default message ("Unsupported git config value and key") in the case that the value/key isn't found instead of asserting they always exist with the `!`.  I think we try to avoid `!` in general just in case..

Reply:
Yep, good point. I will release dugite 2.6.1 with that new function. Thanks!